src/features/schema-manager/schema-builder/openSchemaBuilder.ts

```diff
@@ -44,7 +44,7 @@ async function getOrCreateBuilder(input?: {
   ) {
     builder?.dispose()
 
-    const projectId = input?.projectId ?? (await iamHelpers.askForProjectId())
+    const projectId = input?.projectId ?? (await configVault.requireActiveProjectId())
     if (!projectId) {
       throw new Error(schemaMessage.selectProjectToCreateSchema)
     }
```

Comment:
This one is also no longer relevant. Please remove these and corresponding messages